informatiemodel/rdf/ontologie.ttl

```diff
@@ -100,11 +100,25 @@ zvg:Agent.treed_op_namens a sh:PropertyShape;
 .
 zvg:Appartementsrecht a <http://www.w3.org/2002/07/owl#Class>, sh:NodeShape;
     sh:name "Appartementsrecht";
+    sh:property zvg:Appartementsrecht.aandeelInGemeenschap;
+    sh:property zvg:Appartementsrecht.metVerenigingVanEigenaren;
     sh:targetClass zvg:Appartementsrecht;
     dct:subject <http://kad.zorgeloosvastgoed.nl/id/concept/Appartementsrecht>;
     rdfs:label "Appartementsrecht";
     rdfs:subClassOf zvg:OnroerendeZaak
 .
+zvg:Appartementsrecht.aandeelInGemeenschap a sh:PropertyShape;
+    sh:datatype zvg:Decimal;
+    sh:maxCount "1";
```

Comment:
kunnen we hier een integer van maken?

Reply:
Ik zou verwachten dat het aandeel een breuk is, maar volgens mij is het in dit geval een bedrag. Dus dan is decimal de goede eenheid. Het is geen percentage.